src/ng-xform/modules-import.ts

```diff
@@ -0,0 +1,7 @@
+import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
```

Comment:
Rename this file to `ng-xform.module-imports.ts`

Reply:
OK